highway.tex

```diff
@@ -24,807 +26,1323 @@
 \begin{definition}
-An \emph{endorsement} is a non-vote message between validators that consists of a validator $V$ and the hash of a vote $\lambda$, and is digitally signed by $V$. This endorsement asserts that $V$ did not know of equivocations by the sender of $\lambda$ at the time.
-
-A vote $\lambda$ is said to be \emph{endorsed} if it has endorsements by a collection of validators of total weight more than $2/3$ of the total validator weight.
+A unit $u$ is said to cite another unit $v$ \emph{naively}, denoted $u >_n v$, if $u > v$ and there is no endorsed unit $w$ such that $u > w \geq v$.
 \end{definition}
 
-The protocol will specify that validators create endorsements only of votes sent by other validators that they do not know to be equivocators. We will discuss later the circumstances under which we will actually need them to create endorsements, but for now, we will assume that they create endorsements for every message they receive by a sender that they do not know to be an equivocator. Once an endorsement is created, it can then be shared among the other validators.
-
-We note that this means that an honest validator will never endorse a pair of equivocating votes as by the time they receive the second of these votes, they will know the sender to be an equivocator and thus will not endorse. In fact, exhibiting endorsements of equivocating messages by a single validator is verifiable proof of bad behavior by that validator for which they can be slashed. Furthermore, assuming that the honest validators make up at least $2/3$rds of the total validator weight, this means that no two endorsed votes can be equivocations. This is because for any two endorsed votes $\lambda_1$ and $\lambda_2$, there must be some honest validator that endorsed both of them.
+Using the notion of naive citation, we introduce a validity criterion for units that allows us to avoid equivocation bombs.
 
-We will often think of endorsed votes as being certified by the validators as a whole. While the justifications of other messages might be considered suspect (in that they might contain equivocation bombs), endorsed votes are considered to be more solid.
-\begin{definition}
-A vote $\mu$ is said to cite another vote $\lambda$ \emph{naively}, denoted $\mu \geq_n \lambda$ if $\mu \geq \lambda$ and there is no endorsed vote $\nu$ so that $\mu > \nu \geq \lambda$.
-\end{definition}
+\desc{
+\begin{center}
+    {\bf Limited Naivety Criterion (LNC) } 
+\end{center}
+A unit $u$ by validator $V$ is considered incorrect if there is an equivocation $(v_1, v_2)$ (created by some validator $W$) and two units $u_1, u_2\leq u$ created by $V$ such that 
+$u_1 >_n v_1$  and $u_2 >_n v_2.$
+% $v_1 <_n u_1$  and $v_2 <_n u_2.$
+}
 
-We now introduce a new validity criterion:
+Since, as previously noted, endorsed units, and thus also naive citations are subjective notions, also satisfaction of LNC by a given unit might depend on the local view of a validator.
```

Comment:
Was that previously noted? Maybe that got removed?

It would also be important to point out that a unit can only change from LNC-invalid to LNC-valid and not the other way round, as we gather more endorsements.
I.e.: Not only do we keep _valid_ units in the buffer until we can _cite_ them without violating the LNC ourselves (as already explained below); but we also can request missing endorsements from the sender of a seemingly _invalid_ unit, to prove to us that it complies with the LNC after all.

Reply:
Just after the definition of ENDORSE there is a line "Note that being endorsed is a subjective property as it depends on the local view of a validator.", and that is before the line in question. 
I added the sentence to clarify that the status of a given unit can change only one way.